llms/mlx_lm/utils.py

```diff
@@ -12,28 +13,14 @@
 from transformers import AutoConfig, AutoTokenizer, PreTrainedTokenizer
 
 # Local imports
-from .models import llama, mixtral, olmo, phi2, plamo, qwen, qwen2, stablelm_epoch
 from .tuner.utils import apply_lora_layers
 
 # Constants
-MODEL_MAPPING = {
-    "llama": llama,
-    "mistral": llama,  # mistral is compatible with llama
-    "mixtral": mixtral,
-    "phi": phi2,
-    "stablelm_epoch": stablelm_epoch,
-    "qwen": qwen,
-    "plamo": plamo,
-    "olmo": olmo,
-    "qwen2": qwen2,
+MODEL_REMAPPING = {
+    "mistral": "llama",  # mistral is compatible with llama
+    "phi-msft": "phixtral",
```

Comment:
This may cause some issues because the old phi2 was using this model type. So, if a user tries to load the old phi2 model, it will be mapped to phixtral which won't work. see https://huggingface.co/microsoft/phi-2/blob/5d8f23da6be3205c16c06a9db3f22279ee23dbbf/config.json

Reply:
Hmm, that looks like an old version of phi2. it wouldn't work with mlx lm either way regardless of that remapping. We could try and put a helpful error message when constructing the Phixtral model?

It is too bad that they use the same `model_type`, I think it should have been different..

Reply:
Maybe just mentioned it in the docs? It will error out for missing parameters during the model loading process anyway.